.vscode/extensions.json

```diff
@@ -0,0 +1,12 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+  // Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+  // List of extensions which should be recommended for users of this workspace.
+  "recommendations": [
+    "esbenp.prettier-vscode",
```

Comment:
I think we're using the ESLint prettier plugin, so we shouldn't need this, we should use the ESLint extension.

Reply:
OK, I switched over to the eslint plugin, but I can't get it to actually work as a formatter `onSave`, despite this setting: 
https://github.com/ethereum-optimism/optimism/blob/e5bbe67aa66d1a45277d71a731289834e022dc01/.vscode/settings.json#L3-L5

Do you have it working locally with the eslint plugin?